docs/import/manage_disaster.html

```diff
@@ -104,12 +104,16 @@
       <div>
         <table id="asset-selection-table">
           <thead>
-          <tr id="score-asset-header-row">
+          <tr id="score-asset-header-row"><td>Score Assets</td>
```

Comment:
super nit this is formatted kinda weird

Reply:
Maybe better? Also fixed missing div, I think